### ISSUE TITLE
make apertium-eval-translator part of this package?

### ISSUE THREAD

**Issue by unhammer** [apertium/apertium#31]
http://svn.code.sf.net/p/apertium/svn/trunk/apertium-eval-translator/ is very useful for evaluating progress during development. It'd be nice if it were part of the apertium package, to lower the barrier of entry for using it, and for making sure it isn't left to rot.

**Comment by TinoDidriksen** [comment]
For now, imported as https://github.com/apertium/apertium-eval-translator

**Comment by unhammer** [comment]
It'd also be nice to be able to depend on it for `make check` (to ensure WER doesn't worsen, cf. https://github.com/apertium/apertium-nno-nob/blob/master/t/wer-loss )

**Comment by flammie** [comment]
do you mean like having it installable with pkg-config's and stuff? It'd be pretty trivial autostuff with Makefile.am having bin_SCRIPTS=all perl stuffs and configure.ac and pc.in copypasted from any apertium pkg.

**Comment by unhammer** [comment]
Yeah. Maybe just that one script from https://github.com/apertium/apertium-eval-translator – the rest seem like copies of the same scripts or wrappers around third party tools
